### DISCUSSION TITLE
Is it possible to transfer the installation from usb to vmware?

### DISCUSSION THREAD

**Discussion by crazych1** [PeterSuh-Q3/tinycore-redpill#31] [Q&A] [answered]
I need to transfer the installation from a regular server to esxi, saving all settings and transferring disks with hardware disk forwarding to a virtual machine.

Can you help me with this?

**Comment by PeterSuh-Q3** [comment, marked as answer]
Sorry, this is not my area of expertise.
I have no experience actually handling it.
Please refer to Google or search for guides in the forums below.
https://xpenology.com/forum/
